app/helpers/application_helper.rb

```diff
@@ -62,9 +62,10 @@ def link_to_rss(link_to_feed)
 
   # 1: default shows just the link to help
   # 2: show_text = true: shows "plain text with limited html" and link to help
-  def allowed_html_instructions(show_text = true)
-    (show_text ? h(ts("Plain text with limited HTML")) : "".html_safe) +
-      link_to_help("html-help")
+  def allowed_html_instructions(show_text: true, strip_images: false)
+    (show_text ? t("application_helper.text_limited_html") : "".html_safe) +
+      link_to_help("html-help") +
+      (strip_images ? "<br />".html_safe + t("application_helper.strip_images") : "".html_safe)
```

Comment:
It looks like this never called with `show_text` set to false, so you can remove that parameter.

For when this should have the extra sentence it should just be the sentence directly after the link, no linebreak. Ideally, it would be nice to have one big locale string for both sentences, and switch between that and a different locale strings based on the condition, instead of breaking it up per sentence. And the link should be a variable: https://github.com/otwcode/otwarchive/wiki/Internationalization-%28i18n%29-Standards#user-content-Variable_interpolation

Reply:
Did not see the screenshot as it wasn't linked from the issue. Guess I should have looked around a bit more, sorry.

For the locale bit, do you mean something like:
```yaml
text_limited_html: Plain text with limited HTML %{link}
text_limited_html_strip_images: Plain text with limited HTML %{link} Embedded images (<img> tags) will be displayed as HTML, including the image's source link and any alt text.
```

Reply:
I've run into a bit of difficulty with this, I'm not seeing a neat way to have it escape the text of the string (e.g. where it says "\<img> tags") but _not_ the help link HTML. I could try extending the translate/interpolate function to work better here, or do the escaping/interpolation manually like:
```ruby
html_escape(t(strip_images ? "application_helper.text_limited_html_strip_images" : "application_helper.text_limited_html")).sub("%{help_link}", link_to_help("html-help")).html_safe
```
but otherwise, I think they'd have to be separate strings like they are now. Which would be preferred?

Reply:
It should be hopefully work to use html entities for the lesser than and greater than, so `foo bar &lt;img&gt;` together with the suffix for the locale key `_html` so that the link works

Reply:
Will do, I wasn't sure if that'd cause difficulties for translators.